### PR TITLE
Use Github Actions for PR builds

### PR DESCRIPTION
**Pull request by huwd**
Trello: https://trello.com/c/r22AVt8W/207-switch-the-pr-builds-on-all-forms-to-use-github-actions-not-travis
Trello: https://trello.com/c/TNaiKhFV/228-make-travis-only-build-against-master

### CODE REVIEW

**Review by Rosa-Fox — APPROVED** [review]

**Comment by issyl0** [comment]
Woohoo! Excellent work!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - uses: actions/checkout@v1
+
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.5
+
+    - name: Install Dynamo DB
+      run: |
+        sudo apt-get -yqq install libpq-dev
```
Comment (pixeltrix) [inline .github/workflows/tests.yml]: Is libpq-dev required? Active Record isn't loaded in config/application.rb

Comment (huwd) [reply .github/workflows/tests.yml]: Ah great spot, I'll bin that whole ~line~ section

**Comment by issyl0** [comment]
Thanks for force-,pushing over my Travis commit, Huw. :zany_face: These commits could all be squashed anyway.

**Pull request merged**